mrc/localization/color/kmeans/predicates.py

```diff
@@ -2,7 +2,26 @@
 
 
 class GaussianHSVPredicate:
+    """
+    Class representing predicate based on gaussian distribution.
+
+    Intended for internal usage.
+    """
+
     def __call__(self, hue, range_):
```

Comment:
Wouldn't it be better to pass range as tuple(a, b) or sth similar?

Reply:
Changed